runtime/integration-tests/src/pools/utils/accounts.rs

```diff
@@ -0,0 +1,199 @@
+// Copyright 2021 Centrifuge Foundation (centrifuge.io).
+//
+// This file is part of the Centrifuge chain project.
+// Centrifuge is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version (see http://www.gnu.org/licenses).
+// Centrifuge is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// NOTE: Taken mostly from paritytech-substrate
+
+pub use sp_core::sr25519;
+use sp_core::{
+	sr25519::{Pair, Public, Signature},
+	Pair as PairT,
+};
+use sp_runtime::AccountId32;
+
+/// Set of test accounts.
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
+pub enum Keyring {
+	Admin,
+	TrancheInvestor(u32),
+	Alice,
+	Bob,
+	Charlie,
+	Dave,
+	Eve,
+	Ferdie,
+	Costume(&'static str),
```

Comment:
What does this refer to? Should it be `Custom`?

Reply:
^^ Yes. I will change that.

Reply:
I wil merge it an rename it different PR as otherwise the CI runs again...